### ISSUE TITLE
[Parameter Request] Add -customos steam parameter for unofficial distro usage

### ISSUE THREAD

**Issue by ruineka** [ValveSoftware/steam-for-linux#9118]
There is a growing amount of people using Holoiso and ChimeraOS and I'm starting to see games break on non-deck hardware as they start to add more targeted support to the Steam Deck. Currently we have been forced to use `-steamdeck` to have a usable experience and using just `-gamepadui` has way to many issues to be usable outside of the intended desktop usage.

I don't know all the workings behind the scenes with steam, but in the case of `Sackboy: A Big Adventure`, a recent update broke the game for non-deck hardware because the controller is detected as player 2 instead of player 1.

So long story short the community would like to see something like `-customos` added that doesn't have flags such as `isSteamRunningOnSteamDeck` enabled hopefully fixing situations like this.

I know this is a big ask, but it would mean a lot to see Valve work directly with the community to help us add to the "SteamOS" experience.

**Comment by kisak-valve** [comment]
Hello @ruineka, I don't think this feature request should be evaluated at this time.

We should wait and see how the Steam client behaves around the time that SteamOS 3.X is ready and officially released for general use off of the Steam Deck before looking into something like this. There's a decent chance that this use case will be worked out for that.

To be clear, this isn't a hard NACK, it's just that efforts should be focused on the Steam Deck until the relevant devs are happy with that experience before looking at what would be needed for this.

**Comment by ruineka** [comment]
> Hello @ruineka, I don't think this feature request should be evaluated at this time.
> 
> We should wait and see how the Steam client behaves around the time that SteamOS 3.X is ready and officially released for general use off of the Steam Deck before looking into something like this. There's a decent chance that this use case will be worked out for that.
> 
> To be clear, this isn't a hard NACK, it's just that efforts should be focused on the Steam Deck until the relevant devs are happy with that experience before looking at what would be needed for this.

Thanks for the reply, even if it is a bit disheartening to hear. In the meantime I'll see if I can work out something with Proton that will allow users to disable the Steam Deck check with a launch parameter to work around this. It's been fun pulling apart SteamOS and recreating the experiences for other hardware, but I'm getting a bit fatigued attempting to resolve a never-ending amount of issues. I hope SteamOS for the desktop isn't too far off because the best experience so far has been the Steam Deck experience on PC consoles in the living room.
